data/sql/updates/pending_db_world/blightguard.sql

```diff
@@ -0,0 +1,45 @@
+-- Delete all Alarmed Blightguard, they should be spawned during quest event
+DELETE FROM `creature` WHERE (`id1` = 28745);
+
+-- Update Blightguard AI
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE (`entry`=28603);
+
+-- Add missing skill to Blightguard
+DELETE FROM `creature_template_spell` WHERE (`CreatureID` = 28603);
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES 
+(28603, 0, 33914, 12340);
```

Comment:
Where did you get this from?

You added VerifiedBuild 12340 as the sniff source revision. If it's not from sniff, set this field to 0.